sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 5.10.5-beta.1 (Unreleased)
+
+### Bugs Fixed
+
+- Issue if received message does not have trace context, spans are not created ([#25182](https://github.com/Azure/azure-sdk-for-java/issues/25182))
```

Comment:
```suggestion
- Fixed a bug that when received message does not have trace context, span is not created. ([#25182](https://github.com/Azure/azure-sdk-for-java/issues/25182))
```

Reply:
usually you can take core as example
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/CHANGELOG.md

Reply:
> usually you can take core as example https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/CHANGELOG.md

Thanks. I refer to the example on this page https://azure.github.io/azure-sdk/policies_releases.html#change-logs